src/libs/actions/IOU.js

```diff
@@ -111,8 +111,34 @@ function createIOUSplit(params) {
         });
 }
 
+/**
+ * Settles an IOU Report and then retrieves the iou and chat reports to trigger updates to the UI.
+ */
+function settleIOUReport({
+    chatReportID, reportID, paymentMethodType,
+}) {
+    Onyx.merge(ONYXKEYS.IOU, {loading: true, error: ''});
+    API.PayIOU({
+        reportID,
+        paymentMethodType,
+    })
+        .then((response) => {
+            if (response.jsonCode !== 200) {
+                throw new Error(`Error while settling IOU: ${response.message}`);
+            }
+        })
+        .then(() => fetchChatReportsByIDs([chatReportID]))
+
+        // Any report that is being settled must be open, and must be currently set as open iouReport within the
+        // chatReport object. Therefore, we must also update the chatReport to break this existing link.
```

Comment:
I'm not really sure what this comment means or that it implies. It seems to be missing some context. For example "must be currently set as open iouReport within the chatReport object" seems to reference things that I don't know what they are. "we must also update the chatReport to break this existing link." What is the "existing link" and why does it need broken?

Reply:
Fair point. I've shared a revised comment in [another thread](https://github.com/Expensify/Expensify.cash/pull/2394#discussion_r632612930), so I will resolvee this one.